phones/models.py

```diff
@@ -1,4 +1,241 @@
+from datetime import datetime, timedelta
+import math
+import random
+
+from django.apps import apps
+from django.contrib.auth.models import User
+from django.conf import settings
+from django.core.exceptions import BadRequest
+from django.db.migrations.recorder import MigrationRecorder
 from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch.dispatcher import receiver
+from django.urls import reverse
+
+
+MAX_MINUTES_TO_VERIFY_REAL_PHONE = 5
+
+
+def verification_code_default():
+    return str(math.floor(random.random()*999999)).zfill(6)
+
+
+def verification_sent_date_detaul():
+    return datetime.now()
```

Comment:
Suggestion: `datetime.now(timezone.utc)` for timezone-aware dates. Seeing warnings in server logs:

```
/Users/john/.pyenv/versions/3.9.11/lib/python3.9/site-packages/django/db/models/fields/__init__.py:1416: RuntimeWarning: DateTimeField RealPhone.verification_sent_date received a naive datetime (2022-06-27 20:29:43.103116) while time zone support is active.
  warnings.warn("DateTimeField %s received a naive datetime (%s)"
```